transform/snowflake-dbt/models/finance/arr_transactions.sql

```diff
@@ -46,6 +51,26 @@ with mrr as (
     where ls.issued_date is not null
         and ls.edition in ('Cloud Professional')
         and es.reason is null
+)*/
+
+with mrr as (
+   select
+    customer as stripeid,
+    customers.name as customer_name,
+    subscriptions.id as subscription_id,
+    cws_dns as customer_dns,
+    cws_installation as license_id,
+    subscriptions.created::date as date_joined,
+    current_period_end::date as active_license_end,
+    date_converted_to_paid::date as convert_to_paying,
+    subscriptions.edition as product,
+    canceled_at::date as cancellation_date,
+    subscriptions.status as active_status
+--from {{ref('subscriptions')}}  
+--from {{ source('stripe_raw', 'subscriptions') }}   
+from analytics.stripe.subscriptions 
+left join analytics.stripe.customers on customers.id = subscriptions.customer
+where subscriptions.edition = 'Cloud Professional'
```

Comment:
Thanks for the input here.  I need to refactor my code.  Thought it would be a different product sku name for the annual plan.

Reply:
@ifoukarakis I changed the code.  Instead of identifying the license ids which can be annual post blapi retirement, I am now relying on opportunity ids that were Cloud Professional and Monthly Billing to exclude on ARR if they had a license start date earlier than 6/30/2023.  So when those accounts renew in June 30 as annual the opportunity ids would be new and would be included in ARR